core/compatible.py

```diff
@@ -71,15 +71,15 @@ def check_for_requirements(start_api_server):
     api_config = api_configuration()
     connection_timeout = api_config["api_database_connection_timeout"]
     try:
-        import pymongo
-        import netaddr
-        import flask
-        del netaddr
-        del flask
+        with open(os.path.join(os.getcwd(), 'requirements.txt'), 'r') as requirements:
+            for dependency in requirements:
+                dependency_name = dependency.split('=')[0]
+                exec("import " + dependency_name)
```

Comment:
not sure if it's a good idea, it seems like a code execution vulnerability. e.g. I can add `os; os.system('do something')` to requirements.txt and it will execute it. maybe we should use `__import__(dependency_name)` instead?

Reply:
Yes, that can be an issue. Will try to fix it